wasm-client/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "wasm-client"
```

Comment:
Suggestion: rename to something like "js-verifier" or "light-client-js" because `wasm-bindgen` binds to JS only.